scRNA-seq/00-scRNA-seq_introduction.md

```diff
@@ -1,55 +1,45 @@
 
 ![**Overall view of full-length and tag-based workflows**](diagrams/overview_workflow.png)
 
-### 1) Full-length scRNA-seq  
-*Example:* Smart-seq2 [(Picelli et al. _Nature Protocols._ 2014.)](https://www.nature.com/articles/nprot.2014.006)   
-Cells are physically separated into individual wells of a plate and are
-often also sorted by other means (e.g., Fluorescence Activated Cell Sorting).
+### Full-length scRNA-seq  
+*Example:* Smart-seq2 [(Picelli _et al._ 2014)](https://www.nature.com/articles/nprot.2014.006)    
+Cells are physically separated into individual wells of a plate and are often also sorted by other means (e.g., Fluorescence Activated Cell Sorting).
 Each cell is then sequenced individually and has its own fastq file.
-(This will be two fastq files if this is paired-end sequencing.)
-The data preprocessing steps for these types of scRNA-seq data are similar to
-bulk RNA-seq methods.
+(This will result in two fastq files if this is paired-end sequencing.)
+The data preprocessing steps for these types of scRNA-seq data are similar to bulk RNA-seq methods.
 
 #### Pros:  
 - Can be paired-end sequencing which has less risk for 3' bias.  
-- More complete coverage of transcripts, which may be better for transcript
-discovery purposes.   
+- More complete coverage of transcripts, which may be better for transcript discovery purposes.   
 
 #### Cons:  
 - Is not very efficient (generally 96 cells per plate).  
 - Takes much longer to run (days/weeks depending on sample size).
-- Is a lot more expensive.  
+- Much more expensive.  
 
-### 2) Tag-based scRNA-seq  
-*Example:* 10X Genomics
-[(Zheng et al. _Nat Commun._ 2017.)](https://www.ncbi.nlm.nih.gov/pubmed/28091601)  
+### Tag-based scRNA-seq  
+*Example:* 10X Genomics [(Zheng _et al._ 2017)](https://www.ncbi.nlm.nih.gov/pubmed/28091601)    
 Cells are separated by emulsion/droplets, and individual cells are given barcodes.
 Everything is then sequenced.
-These types of methods, because they are newer, are more likely to have
-Unique Molecular Identifiers (UMIs)
-[(Islam et al. _Nature Methods._ 2014.)](http://www.nature.com/doifinder/10.1038/nmeth.2772),
-which allow you to better control for PCR amplification errors and biases.
-Individual samples have two fastq files: one for the cell barcodes
-and another with the individual reads.
+These types of methods, because they are newer, are more likely to have Unique Molecular Identifiers (UMIs) [(Islam _et al._ 2014)](http://www.nature.com/doifinder/10.1038/nmeth.2772). 
+These are additional barcodes added before amplification which allow the identification of PCR duplicates, allowing control for PCR amplification errors and biases.
+Individual samples have two fastq files: one for the cell barcodes and another with the individual reads.
 
 #### Pros:  
 - Can run potentially millions of cells at once.   
-- A lot quicker computing wise.  
+- Much less computationally demaning.
```

Comment:
```suggestion
- Much less computationally demanding.
```

Reply:
Good catch, but how did you know I didn't mean "demeaning"?

Reply:
I did have an internal back and forth about whether or not you meant "demeaning" or "demanding" here but context clues convinced be to go with the latter 😏